client/app/scripts/charts/nodes-grid.js

```diff
@@ -0,0 +1,147 @@
+/* eslint react/jsx-no-bind: "off", no-multi-comp: "off" */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { List as makeList, Map as makeMap } from 'immutable';
+import NodeDetailsTable from '../components/node-details/node-details-table';
+import { clickNode, sortOrderChanged, clickPauseUpdate,
+  clickResumeUpdate } from '../actions/app-actions';
+
+import { getNodeColor } from '../utils/color-utils';
+
+
+const IGNORED_COLUMNS = ['docker_container_ports', 'docker_container_id', 'docker_image_id',
```

Comment:
What's important and what can be ignored, should be determined by the backend. The current metadata field `priority` is not sufficient, though. Looks like we need another field.